Source/driver/Castro.cpp

```diff
@@ -753,12 +755,12 @@ Castro::initMFs()
 	mass_fluxes[dir].reset(new MultiFab(get_new_data(State_Type).boxArray(), dmap, 1, 0));
 
 #if (BL_SPACEDIM <= 2)
-    if (!Geometry::IsCartesian())
+    if (!Geom().IsCartesian())
```

Comment:
What's the difference between DefaultGeometry() and Geom()?

Reply:
`Geom()` is a member function of `AmrLevel`, whereas `DefaultGeometry` is a free function in amrex namespace.  For `IsCartesian()`, both are fine.  But for things like `CellSize`, one has to use `Geom()` or `Geom(int level)` in `Amr` class.